### PR TITLE
Adding a section on moving to production with cert-manager

### PR DESCRIPTION
**Pull request by joshuacox**
Just adding some explicit instruction on moving to production.

### CODE REVIEW

**Comment by joshuacox** [comment]
signed the CLA just now

File: content/docs/examples/advanced-gateways/ingress-certmgr/index.md [hunk 1]
```diff
@@ -176,3 +176,56 @@ Hello version: v1, instance: helloworld-5d498979b6-jp2mf
...
+    - http01:
+        ingressClass: istio
+      domains:
+      - $INGRESS_DOMAIN
+---
+EOF
+{{< /text >}}
+
+
+```
```
Comment (geeknoid) [inline content/docs/examples/advanced-gateways/ingress-certmgr/index.md]: For all uses of triple-backticks, please switch to use our {{< text >}} construct. And for any command-lines, start the text block with a $

Thanks.

File: content/docs/examples/advanced-gateways/ingress-certmgr/index.md [hunk 2]
```diff
@@ -176,3 +176,56 @@ Hello version: v1, instance: helloworld-5d498979b6-jp2mf
...
+
+Now delete the secret to force cert-manager to request a new cert from the production issuer:
+
+```
+kubectl delete secret -n istio-system ingress-cert-staging
+```
+
+And watch that cert for a successful issuance:
```
Comment (geeknoid) [inline content/docs/examples/advanced-gateways/ingress-certmgr/index.md]: here and elsewhere: cert -> certificate

**Comment by geeknoid** [comment]
If you rebase, that'll get rid of the bad link reported by the CI system.

File: content/docs/examples/advanced-gateways/ingress-certmgr/index.md (hunk 2 above)
Comment (joshuacox) [reply content/docs/examples/advanced-gateways/ingress-certmgr/index.md]: corrected that and the links

File: content/docs/examples/advanced-gateways/ingress-certmgr/index.md (hunk 1 above)
Comment (joshuacox) [reply content/docs/examples/advanced-gateways/ingress-certmgr/index.md]: and removed the triple backticks and replaced with the text block convention

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit a5fc0e105fd155bf3d37e4e728edefdd5fef65d4

https://app.netlify.com/sites/preliminary-istio/deploys/5cbf3b6dd412630007888e08

**Comment by joshuacox** [comment]
ok where is the documentation for this quoting method?

I don't see it here:
https://help.github.com/en/articles/basic-writing-and-formatting-syntax

File: content/docs/examples/advanced-gateways/ingress-certmgr/index.md [hunk 3]
```diff
@@ -176,3 +176,56 @@ Hello version: v1, instance: helloworld-5d498979b6-jp2mf
...
+## Moving to production from staging
+
+Now to switch to the production `letsencrypt` issuer.  First we'll reapply the certificate.
+
+{{< text bash >}}
+$ cat <<EOF | kubectl apply -f -
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Certificate
+metadata:
+  name: ingress-cert-staging
```
Comment (philrud) [inline content/docs/examples/advanced-gateways/ingress-certmgr/index.md]: We should probably get rid of "-staging" suffix in the name/secretName everywhere in this example if we're going to reuse same resource/secret for both staging and prod. Or use another Cert/Secret pair for prod if it is possible to hotswap them without having to restart ingressgateway.

Comment (joshuacox) [reply content/docs/examples/advanced-gateways/ingress-certmgr/index.md]: removing

**Comment by geeknoid** [comment]
https://istio.io/create has got info on writing Istio docs.

**Comment by joshuacox** [comment]
ok finally passed the gauntlet of lint in the CI

File: content/docs/examples/advanced-gateways/ingress-certmgr/index.md [hunk 4]
```diff
@@ -176,3 +176,54 @@ Hello version: v1, instance: helloworld-5d498979b6-jp2mf
...
+    config:
+    - http01:
+        ingressClass: istio
+      domains:
+      - $INGRESS_DOMAIN
+---
+EOF
+{{< /text >}}
+
+{{< text java>}}
```
Comment (geeknoid) [inline content/docs/examples/advanced-gateways/ingress-certmgr/index.md]: Change to {{< text plain >}}

File: content/docs/examples/advanced-gateways/ingress-certmgr/index.md [hunk 5]
```diff
@@ -176,3 +176,54 @@ Hello version: v1, instance: helloworld-5d498979b6-jp2mf
...
+
+And watch that cert for a successful issuance:
+
+{{< text bash >}}
+$ watch -n1 kubectl describe cert ingress-cert -n istio-system
+{{< /text >}}
+
+you should see something like:
+
+{{< text java>}}
```
Comment (geeknoid) [inline content/docs/examples/advanced-gateways/ingress-certmgr/index.md]: also {{< text plain >}}

File: content/boilerplates/notes/0.8.md [hunk 6]
```diff
@@ -5,7 +5,7 @@
 deployment [use-cases](/docs/tasks/traffic-management/). For folks upgrading from earlier releases there is a
 [migration guide](/docs/setup/kubernetes/upgrade/) and a conversion tool built into `istioctl` to help convert your configuration from the old model.
 
-- **Streaming Envoy configuration**. By default Pilot now streams configuration to Envoy using its [ADS API](https://github.com/envoyproxy/data-plane-api/blob/master/XDS_PROTOCOL.md). This new approach increases effective scalability, reduces rollout delay and should eliminate spurious 404 errors.
+- **Streaming Envoy configuration**. By default Pilot now streams configuration to Envoy using its [ADS API](https://github.com/envoyproxy/data-plane-api/blob/master/xds_protocol.rst). This new approach increases effective scalability, reduces rollout delay and should eliminate spurious 404 errors.
```
Comment (geeknoid) [inline content/boilerplates/notes/0.8.md]: This change and the corresponding Chinese change have already been introduced to the source base this morning. So if you rebase on the latest bits, you won't need these two changes in your commit.

**Comment by joshuacox** [comment]
@geeknoid there is no reason to keep all those typo fix commits so I just squashed and took the opportunity to squash and sign at the same time.

**Pull request merged**